### ISSUE TITLE
Unable to process formData 

### ISSUE THREAD

**Issue by se-achiu** [geoffrich/svelte-adapter-azure-swa#115]
There is an issue where azure swa managed function sk_render doesn't able to received file from page.svelte which is using multipart/form-data and page.server.ts which have svelte actions that takes the file from formdata that contains a file.  It is working on local development, but once deploy on SWA, the app running with the sk_render api on the managed function app is no longer able to get the formdata.

**Comment by geoffrich** [comment]
Hi! Can you provide a repo with a minimal reproduction?

**Comment by se-achiu** [comment]
Sorry for the late respond, here is the repo: https://github.com/se-achiu/svelte-adapter-azure-swa-multipart-test
when running locally, it will returns the multipart formdata correctly.  But once deployed on Azure SWA, it will not get the multipart-formdata content-type: https://ashy-moss-07ea8a60f.2.azurestaticapps.net/

**Comment by vd-aloker** [comment]
There is definitely something going on with uploads to Azure SWA. I'm having massive issues with uploaded files being corrupted (lots of UTF-8 replacement chars in the file, ie. "EF BF BD") when deployed to SWA or running in the SWA CLI whereas it runs fine on node deployments. I'm trying to come up with a reproduction repo.

**Comment by vd-aloker** [comment]
Here's a repo https://github.com/aloker/swa-file-issues

The form prints the size of the uploaded file. Here's a screenshot of a JPEG file I've uploaded - left side: node/preview, right side: SWA CLI (but it's identical on Azure SWA). Notice the difference in size. The file content is corrupted as mentioned in my previous post.

![image](https://user-images.githubusercontent.com/101794077/225308834-4c028083-e7d9-4957-8864-cad12cc19cbe.png)

**Comment by vd-aloker** [comment]
This is probably the offending part: 
https://github.com/geoffrich/svelte-adapter-azure-swa/blob/44ebb6c9c8dfc6254c343ffba39552424a759a7b/files/entry.js#L73

If the body contains multipart formdata, `rawBody` contains the binary data of the file (separated by the boundary). This certainly can't be parsed as UTF-8 and explains the error. 

I've created a a PR to fix this.

My excuses if my issue and the one of @se-achiu are not related.

**Comment by geoffrich** [comment]
I just released svelte-adapter-azure-swa@0.14 with @aloker's fix. @se-achiu, can you check if this fixes your issue as well?

**Comment by geoffrich** [comment]
Briefly started looking into this -- it seems that multipart form data and Azure functions is a common stumbling block. Haven't figured out what the best solution is here. Random resources I've stumbled on:

- https://www.sergevandenoever.nl/Processing-multipart-form-data-in-nodejs-azure-function-with-httptrigger/
- https://stackoverflow.com/questions/50526133/how-to-upload-a-file-using-node-js-function-in-azure
- https://github.com/Azure/azure-functions-host/issues/2009
- https://social.msdn.microsoft.com/Forums/sqlserver/en-US/551debce-57f0-43f8-8447-c00bd77ba37a/httptrigger-with-multipart-formdata?forum=AzureFunctions

**Comment by geoffrich** [comment]
We might need to use a library [like this](https://www.npmjs.com/package/parse-multipart-data) to parse the data to a Buffer.

**Comment by geoffrich** [comment]
Actually, wrong rabbit hole - undici (which is used by SvelteKit to polyfill Request/Response) should be handling this for us. I updated my copy of the issue repro to the latest version of SvelteKit (1.12) and it seemed to fix the issue (screenshot below).

@se-achiu can you install the latest SvelteKit and let me know if that fixes it? Looks like your repro is on SvelteKit 1.8.3, see if it's still there with 1.12 (or later).

https://polite-desert-00b80111e-130.westus2.2.azurestaticapps.net/file-issue

<img width="1171" alt="image" src="https://user-images.githubusercontent.com/4992896/226143833-5ad8e292-140a-49e1-82f9-2d66380c0cae.png">

**Comment by se-achiu** [comment]
@geoffrich, As I was trying to determined the exact version that fixes the issue, I upgrade sveltekit to 1.8.2 thinking that 1.8.3 was the one, and the formdata was parse correctly, so I kept downgrade back to 1.6.0, then 1.5.1, and back to the original 1.5.0.  And they all were fixed, and then I downgrade svelte-adapter-azure-swa from 0.14.0 to 0.13.0.  And it showed the utf-8 issue that @vd-aloker resolved.
0.13.0
![image](https://user-images.githubusercontent.com/44265811/226430690-8bc74d74-5b3b-492c-a2ee-4067c54c4440.png)
0.14.0
![image](https://user-images.githubusercontent.com/44265811/226430887-a58e4015-d0ec-4927-9616-a85fd874d878.png)

So I wasn't sure what fixes it.  As previously, it was showing the ContentType text/plain.
